packaging/brew/ignite.rb

```diff
@@ -0,0 +1,22 @@
+class Ignite < Formula
+  desc "Build, launch, and maintain any crypto application with Ignite CLI"
+  homepage "https://github.com/ignite/cli"
+  url "https://github.com/ignite/cli/archive/refs/tags/v28.2.0.tar.gz"
+  sha256 "556f953fd7f922354dea64e7b3dade5dd75b3f62ece93167e2ba126cac27602e"
+  license "Apache-2.0"
+
+  depends_on "go"
+  depends_on "node"
```

Comment:
Wouldn't be good to use specific versions?

Reply:
I've checked other formulae and that isn't something that is done.